modules/graphql_api_endpoint/api_endpoint.tf

```diff
@@ -0,0 +1,97 @@
+resource "aws_api_gateway_resource" "graphql_endpoint" {
```

Comment:
Have all the resources that support tagging got tags assigned to them

Reply:
See [previous comment](https://github.com/nationalarchives/tdr-prototype-terraform/pull/9/files#r318973847).